api/lovr/graphics/setDepthTest.lua

```diff
@@ -8,7 +8,7 @@ return {
     {
       name = 'compareMode',
       type = 'CompareMode',
-      default = 'nil',
+      default = [['lequal']],
```

Comment:
The default is actually `nil` for this.  "Default" here means the default value for the argument itself, it isn't referring to the "default state value".  Example:

```lua
-- Since compareMode isn't specified, nil will be used, disabling the depth test:
lovr.graphics.setDepthTest()

-- Should print 'nil', indicating that the depth test is disabled, not 'lequal', even
-- though the state starts out at 'lequal'
print(lovr.graphics.getDepthTest())
```

The bit in the notes section mentions the default value for the state.  That's at least the reasoning behind how the docs are structured today, and I'm open to feedback on ways to make it more clear.